### DISCUSSION TITLE
Why footer is not applying on my website?

### DISCUSSION THREAD

**Discussion by spencermaddox2** [twbs/bootstrap#37519] [General]
I have a website based on SA Fakaza | [Download African songs](https://safakaza.com/) Mp3, Videos, Amapiano, Gqom, hiphopza, lyrics Fakaza Music Zamusic.
My website footer contains my website details but all my details are misplaced and footer is not working properly, sometimes this list in a proper way but sometimes all my lists are moved from there exact place, Kindly fix this issue or help me, Thanks.

**Comment by iam-AadarshSingh** [comment]
Have you connected bootstrap to your file . if not then try that, or if you have already done. then can I see your code ?
